py/server/deephaven/table.py

```diff
@@ -2084,6 +2084,29 @@ def slice(self, start: int, stop: int) -> Table:
         except Exception as e:
             raise DHError(e, "table slice operation failed.") from e
 
+    def slice_pct(self, start_pct: float, end_pct: float) -> Table:
+        """Extracts a subset of a table by row percentages.
+
+        Returns a subset of table in the range [floor(start_pct * size_of_table), floor(end_pct * size_of_table)).
+        For example, for a table of size 10, slice_pct(0.1, 0.7) will return a subset from the second row to the seventh
+        row. Similarly, slice_pct(0, 1) would return the entire table (because row positions run from 0 to size-1).
+        The percentage arguments must be in range [0,1], otherwise the function returns an error.
```

Comment:
```suggestion
        row. Similarly, slice_pct(0, 1) would return the entire table (because row positions run from 0 to size - 1).
        The percentage arguments must be in range [0, 1], otherwise the function returns an error.
```